tests/src/test/java/alluxio/server/worker/PagedBlockStoreIntegrationTest.java

```diff
@@ -123,7 +123,7 @@ public void localPageStorePreservesPagesAfterRestart() throws Exception {
   @LocalAlluxioClusterResource.Config(confParams = {
       PropertyKey.Name.WORKER_PAGE_STORE_PAGE_SIZE, "64",
       PropertyKey.Name.USER_STREAMING_READER_CHUNK_SIZE_BYTES, "16",
-      PropertyKey.Name.WORKER_NETWORK_READER_BUFFER_POOLED, "false"
```

Comment:
As the above discussion, we should keep the WORKER_NETWORK_READER_BUFFER_POOLED besides.